WickedEngine/wiRenderPath3D.cpp

```diff
@@ -1079,7 +1079,7 @@ namespace wi
 				RenderPassImage rp[] = {
 					RenderPassImage::RenderTarget(
 						&rtReflection,
-						RenderPassImage::LoadOp::DONTCARE,
+						RenderPassImage::LoadOp::CLEAR,
```

Comment:
We only want to clear if sky is not rendered, otherwise it is more optimal to not clear.